src/Polly.Shared/PolicyResult.cs

```diff
@@ -117,7 +143,13 @@ internal static PolicyResult<TResult> Failure(Exception exception, ExceptionType
                 context);
         }
 
-        internal static PolicyResult<TResult> Failure(TResult handledResult, Context context)
+        /// <summary>
+        /// Builds a failed <see cref="PolicyResult" />
+        /// </summary>
+        /// <param name="handledResult">The handled result</param>
```

Comment:
((suggest change to))

`The result returned by execution through the policy, which was treated as a handled failure`